packages/localize/types/LocalizeMixinTypes.d.ts

```diff
@@ -1,4 +1,5 @@
 import { Constructor } from '@open-wc/dedupe-mixin';
+import { LitElement } from 'lit-element';
```

Comment:
import from `@lion/core` so we control the version, in case lit-element releases an update that we need to hold off on because of incompatibilities.